common/history/countries/0_dvg_ameriga.txt

```diff
@@ -128,13 +128,16 @@
 		add_technology_researched = colonization
 		
 		activate_law = law_type:law_national_supremacy
-		activate_law = law_type:law_interventionism
 		activate_law = law_type:law_land_based_taxation
 		activate_law = law_type:law_colonial_resettlement
 		activate_law = law_type:law_religious_schools
 		activate_law = law_type:law_censorship
 		activate_law = law_type:law_migration_controls
-		
+		activate_law = law_type:law_women_own_property
+		activate_law = law_type:law_command_economy
```

Comment:
this creates an error at game start obviously, thanks to Vic 3 locking laws behind techs, but it's the best way to represent the Incan economy at the moment (TWT was state capitalism at game start in Vic 2 for example). Unless anyone else has a better suggestion, this is the only way to go for now.

Reply:
We should prob avoid giving laws that are behind techs. Not sure if Inca actually is a Command Economy? I don't think there is anything that represents state capitalism as in Vic2. Maybe Interventionism comes closest from the starting laws?

Reply:
I don't like it either, but unfortunately it's the best representation currently in the game, short of making a snowflake law just for the Incas, which I don't like either. Interventionism doesn't mandate government owned, and the Incan state owned all the land and all means of production, and would move people around to work where needed on farms or infrastructure projects or the like using the mita labour draft, which was used in place of a system of taxation. It's been described as proto-socialist by some historians, and a lot of the DoDFF flavour events are about recognizing the similarities between the Incan economic model and the socialist one, and modernizing the system as a preventative against socialist agitation.